src/bin/index.ts

```diff
@@ -106,9 +113,16 @@ parseCommandLineArguments()
         return accountReadinessCommand({ credentialProvider: awsCredentialProviderChain(profile), region });
       }
       case Commands.New: {
-        const { init, app, stack, yamlTemplateLocation, stage } = argv;
+        const { init, app, stack, yamlTemplateLocation, stage, packageManager } = argv;
         const stages = stage.map((_) => (_ as string).toUpperCase());
-        return newCdkProject({ init, app, stack, yamlTemplateLocation, stages });
+        return newCdkProject({
+          init,
+          app,
+          stack,
+          yamlTemplateLocation,
+          stages,
+          packageManager: packageManager as PackageManager,
```

Comment:
Am I right in thinking that this is a type-cast and there's no validation for this field like there is for, e.g., some of the paths? So if someone mis-spelled the package manager, or actually even typed `pnpm`, that string would make its way through to the `runTask` function?

Reply:
Yep you're correct. We're using [yargs's choices](https://yargs.js.org/docs/#api-reference-choiceskey-choices) to ensure the value of `packageManager` is correct by this point.

Reply:
Oh that's a nice feature. Ok so that'll catch it before it gets to this point 👍